src/components/Amount.tsx

```diff
@@ -30,12 +32,34 @@ const Amount = ({ value, className }: AmountProps) => {
       general: { discreetMode }
     }
   } = useGlobalContext()
+  let integralPart = ''
+  let fractionalPart = ''
+
+  if (!discreetMode && value !== undefined) {
+    const abbreviatedAmountParts = abbreviateAmount(value).split('.')
+    integralPart = abbreviatedAmountParts[0]
+    fractionalPart = abbreviatedAmountParts[1]
+  }
 
   return (
-    <span className={className}>{`${
-      discreetMode ? '•••' : value !== undefined ? abbreviateAmount(value) : '-'
-    } ℵ`}</span>
+    <span className={className}>
+      {discreetMode ? (
+        '•••'
+      ) : value !== undefined ? (
+        <>
+          <span>{integralPart}</span>
+          <Decimals>.{fractionalPart}</Decimals>
+        </>
+      ) : (
+        '-'
+      )}
+      {' ℵ'}
+    </span>
   )
 }
 
+const Decimals = styled.span`
+  /* font-size: 0.85em; */
```

Comment:
@mvaivre, I just commented it out so that we can experiment with this later on without reinventing the wheel that I just invented 😅